### PR TITLE
Pr/meta update

### PR DESCRIPTION
**Pull request by Onset**

### CODE REVIEW

File: theme/views/@layout.latte [hunk 1]
```diff
@@ -1,27 +1,25 @@
 <!DOCTYPE html>
 
-{default $description = ''}
-{default $keywords = ''}
-{default $activeLang = 'cs'}
+{default $activeLang = explode('_', get_locale())[0]}
```
Comment (ViliamKopecky) [inline theme/views/@layout.latte]: This would be nicer in some util function like `getActiveLangCode`

File: theme/views/@layout.latte [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
 
-<html>
+<html lang="{$activeLang}">
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 	<meta name="viewport" content="width=device-width,initial-scale=1">
 
-	<title>{site_title()}</title>
+	{site_title()}
```
Comment (ViliamKopecky) [inline theme/views/@layout.latte]: This might generate something else than `<title>` tag?

File: theme/views/@layout.latte [hunk 3]
```diff
@@ -1,27 +1,25 @@
...
 
-	<meta name="description" content="{$description}">
-	<meta name="keywords" content="{$keywords}">
+	<meta name="description" content="{get_bloginfo('description')|noescape}">
```
Comment (ViliamKopecky) [inline theme/views/@layout.latte]: Why `noescape`‽

File: theme/views/@layout.latte (hunk 1 above)
Comment (Onset) [inline theme/views/@layout.latte]: Done.
https://github.com/manGoweb/MangoPress/pull/64/commits/0eb049039db12f0b852c839f3059982542c79097

File: theme/views/@layout.latte (hunk 3 above)
Comment (Onset) [inline theme/views/@layout.latte]: Latte doesn't know that it's already escaped by WordPress.

Comment (ViliamKopecky) [inline theme/views/@layout.latte]: omg, alright :D

File: theme/views/@layout.latte (hunk 2 above)
Comment (Onset) [inline theme/views/@layout.latte]: AFAIK it's much easier to create Nette Html element with wrapping `<title />` tag.
https://github.com/manGoweb/MangoPress/pull/64/files/a62fc3d84fa316ce8c5bdd4999a281703e3ae6cf#diff-6ecff5c8ded1ef98dec31e6b03d293d1R2

Comment (ViliamKopecky) [inline theme/views/@layout.latte]: What about `Html::el()->setHtml('html content without a wrapper tag')`

Comment (Onset) [inline theme/views/@layout.latte]: :+1: Nice. Done.
https://github.com/manGoweb/MangoPress/pull/64/commits/fd77e172ecd417b66c415c4286e98eccce976370#diff-6ecff5c8ded1ef98dec31e6b03d293d1R5

**Pull request merged**